### PR TITLE
docs: add alchemy signer passkey docs

### PR DESCRIPTION
**Pull request by avasisht23**
# Pull Request Checklist

- [ ] Did you add new tests and confirm existing tests pass? (`yarn test`)
- [ ] Did you update relevant docs? (docs are found in the `site` folder, and guidelines for updating/adding docs can be found in the [contribution guide](https://github.com/alchemyplatform/aa-sdk/blob/main/CONTRIBUTING.md))
- [ ] Do your commits follow the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) standard?
- [ ] Does your PR title also follow the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) standard?
- [ ] If you have a breaking change, is it [correctly reflected in your commit message](https://www.conventionalcommits.org/en/v1.0.0/#examples)? (e.g. `feat!: breaking change`)
- [ ] Did you run lint (`yarn lint:check`) and fix any issues? (`yarn lint:write`)
- [ ] Did you follow the [contribution guidelines](https://github.com/alchemyplatform/aa-sdk/blob/main/CONTRIBUTING.md)?

---

## PR-Codex overview
The focus of this PR is to enhance passkey authentication and signup functionality in the Alchemy Signer. 

### Detailed summary
- Updated paths and imports for passkey authentication and signup
- Added passkey signup and authentication components
- Improved meta tags for passkey signup and authentication pages
- Updated sidebar links for Alchemy Signer pages

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: site/signers/alchemy-signer.md
Comment (avasisht23) [inline site/signers/alchemy-signer.md:1]: we have this already - this is a dupe

File: site/signers/alchemy-signer/passkey-auth.md
Comment (avasisht23) [inline site/signers/alchemy-signer/passkey-auth.md:1]: we have this already - this is a dupe so replaced

File: site/signers/alchemy-signer/manage-user-sessions.md
Comment (avasisht23) [inline site/signers/alchemy-signer/manage-user-sessions.md:1]: dangling doc, moved to "using alchemy signer" section

File: site/using-smart-accounts/export-private-key.md
Comment (avasisht23) [inline site/using-smart-accounts/export-private-key.md:1]: this is a dupe

**Review by moldy530 — APPROVED** [review]

**Pull request merged**